src/RecoveryServices/RecoveryServices.SiteRecovery/Models/2016-08-10/ASRPSObjects.cs

```diff
@@ -365,6 +365,7 @@ public ASRAzureToAzureVmSyncedConfigDetails_2016_08_10(AzureToAzureVmSyncedConfi
         /// <summary>
         /// Gets or sets the Azure role assignments.
         /// </summary>
+        [Obsolete("This property will be removed in a future release.")]
```

Comment:
Do not use the Obsolete attribute.  Please use the existing breakign change attributes as described here: https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-attribute-help.md#cmdletoutputbreakingchangeattribute  APply to all applications of the Obsolete attribute